### DISCUSSION TITLE
How can I pass a custom step value to the slider?

### DISCUSSION THREAD

**Discussion by graf-andreev** [andreruffert/rangeslider.js#376] [Q&A] [answered]
Hello everyone. How can I pass a custom step value to the slider? For example, I need steps of 15, 30, 60, 90.  

This code doesn't work.

$('input[type="range"]').rangeslider({
  step: [15, 30, 60, 90]
});

**Comment by andreruffert** [comment, marked as answer]
@graf-andreev based on the [docs](https://rangeslider.js.org/#options) the `step` option is not supported that's why your described code above is not working.

Have a look at this [codepen](https://codepen.io/andreruffert/pen/dOJaZE).

It describes an approach of value mapping. In your case this would look something like this:

```html
<input type="range" min="0" max="3" step="1" value="0">
```
```js
const valueMapping = [15, 30, 60, 90]
```

✌️
